src/mame/sinclair/tsconf_m.cpp

```diff
@@ -451,6 +478,8 @@ void tsconf_state::cram_write16(offs_t offset, u16 data)
 
 void tsconf_state::sfile_write16(offs_t offset, u16 data)
 {
+	m_sprites_cache.clear();
```

Comment:
Should this also be cleared in `device_post_load`?  Loading a saved state will cause the contents of memory to change under it.